controllers/meController.js

```diff
@@ -0,0 +1,35 @@
+var db = require('../models');
+
+function index(req, res){
```

Comment:
It seems like you wrote this controller code, but never used it--that's a bad sign, and tells me that you're writing code without knowing how it'll fit into your overall app.